lib/teleterm/api/proto/v1/service.proto

```diff
@@ -33,11 +34,29 @@ service TerminalService {
   // ListLeafClusters lists leaf clusters
   rpc ListLeafClusters(ListLeafClustersRequest) returns (ListClustersResponse);
   // ListDatabases lists databases
-  rpc ListDatabases(ListDatabasesRequest) returns (ListDatabasesResponse);
+  rpc GetAllDatabases(GetAllDatabasesRequest) returns (GetAllDatabasesResponse);
+  // ListDatabaseUsers lists allowed users for the given database based on the role set.
+  rpc GetDatabases(GetDatabasesRequest) returns (GetDatabasesResponse);
   // ListDatabaseUsers lists allowed users for the given database based on the role set.
   rpc ListDatabaseUsers(ListDatabaseUsersRequest) returns (ListDatabaseUsersResponse);
-  // ListServers lists servers
-  rpc ListServers(ListServersRequest) returns (ListServersResponse);
+  // GetAllServers returns all servers
+  rpc GetAllServers(GetAllServersRequest) returns (GetAllServersResponse);
+  // GetServers returns filtered, sorted, and paginated servers
+  rpc GetServers(GetServersRequest) returns (GetServersResponse);
+  // GetAccessRequests lists filtered AccessRequests
+  rpc GetAccessRequests(GetAccessRequestsRequest) returns (GetAccessRequestsResponse);
+  // GetAccessRequest retreives a single Access Request
+  rpc GetAccessRequest(GetAccessRequestsRequest) returns (GetAccessRequestResponse);
```

Comment:
Do they have to share the same request type? Is there a situation where we want to call `GetAccessRequests` with a specific request ID?

Reply:
This is because `authClient` only has a plural `GetAccessRequests`. I figured `clusters_access_request.go` could have a single Get like the authclient and treat ID as essentially a filter. `handler_access_requests` has a distinct method to return just 1 request from the ID.

Reply:
I ended up moving into separate RPCs. A couple other things was I removed the `user` and `RequestState` params because we don't do server side searching for AccessRequests on web or Connect so they are dead params anyway. `GetAccessRequest` now accepts an ID/cluster_uri and `GetAccessRequests` only accepts the cluster_uri. The service now checks for existence of `req.Id` instead of cluster method (also in DeleteRequests). So now, the service has two RPCs with separate requests that are used to just pass `types.AccessRequestsFilter` to the cluster method